internal/services/appservice/helpers/web_app_slot_schema.go

```diff
@@ -666,6 +666,10 @@ func (s *SiteConfigLinuxWebAppSlot) ExpandForCreate(appSettings map[string]strin
 		expanded.RemoteDebuggingVersion = pointer.To(s.RemoteDebuggingVersion)
 	}
 
+	if s.WorkerCount != 0 {
+		expanded.NumberOfWorkers = pointer.To(int32(s.WorkerCount))
```

Comment:
Due to a recent API update, this is now an `int64` so would need to be:
```suggestion
		expanded.NumberOfWorkers = pointer.To(s.WorkerCount)
```

Reply:
@jackofallops  thank you for your reply. I've updated the PR with your suggestions.